### PR TITLE
feat: enable to send controller information via aap credential type

### PR DESCRIPTION
**Pull request by hsong-rh**
https://issues.redhat.com/browse/AAP-22061

### CODE REVIEW

File: src/aap_eda/wsapi/consumers.py [hunk 1]
```diff
@@ -117,13 +117,12 @@ async def handle_workers(self, message: WorkerMessage):
...
-        awx_token = await self.get_awx_token(message)
-        if awx_token:
-            controller_info = ControllerInfo(
-                url=settings.EDA_CONTROLLER_URL,
-                token=awx_token,
-                ssl_verify=settings.EDA_CONTROLLER_SSL_VERIFY,
-            )
+
+        controller_info = await self.get_controller_info(message)
+        logger.error(
```
Comment (mkanoor) [inline src/aap_eda/wsapi/consumers.py:122]: @hsong-rh This might not be needed

**Comment by mkanoor** [comment]
@hsong-rh can we add some tests for this

**Comment by hsong-rh** [comment]
> @hsong-rh can we add some tests for this

Test cases added

File: src/aap_eda/wsapi/consumers.py [hunk 2]
```diff
@@ -345,6 +341,50 @@ def get_awx_token(self, message: WorkerMessage) -> tp.Optional[str]:
...
+        )
+        for eda_credential in parent.eda_credentials.all():
+            if eda_credential.credential_type.id == aap_credential_type.id:
+                inputs = yaml.safe_load(
+                    eda_credential.inputs.get_secret_value()
+                )
+                return ControllerInfo(
+                    url=inputs["host"],
+                    token=inputs.get("oauth_token", ""),
+                    ssl_verify=inputs.get("ssl_verify", "yes"),
```
Comment (mkanoor) [inline src/aap_eda/wsapi/consumers.py:381]: @hsong-rh The RH AAP calls the attribute verify_ssl not ssl_verify. https://github.com/ansible/eda-server/blob/963f5a0807815a05204ce9fe50c5669d60a7fe4f/src/aap_eda/core/management/commands/create_initial_data.py#L270

Comment (hsong-rh) [reply src/aap_eda/wsapi/consumers.py]: https://github.com/ansible/eda-server/blob/main/src/aap_eda/wsapi/messages.py#L80
what we defined as string is also for future certificate purpose

**Review by mkanoor — APPROVED** [review]

**Review by bzwei — APPROVED** [review]

**Pull request merged**